README.md

```diff
@@ -287,9 +290,9 @@ titleTemplate = '%s | Next SEO';
 
 #### No Index
 
-Setting this to `true` will set `no-index`, `no-follow`. This works on a page by page basis.
+Setting this to `true` will set `noindex,follow` (to set `nofollow`, please refer to [`no-follow`](#noFollow)). This works on a page by page basis. This property works in tandem with the `no-follow` property and together they populate the `robots` and `googlebot` meta tags.
 
-So this property is a little different than all the others in the sense that setting it as a default does not work as expected. This is due to the fact Next SEO already has a default of `index` as it is an SEO plugin after all. So if you want to globally set `noindex` please see [dangerouslySetAllPagesToNoIndex](#dangerouslySetAllPagesToNoIndex)
+**Note:** The `no-index` and the [`no-follow`](#noFollow) properties are a little different than all the others in the sense that setting them as a default does not work as expected. This is due to the fact Next SEO already has a default of `index,follow` because `next-seo` is a SEO plugin after all. So if you want to globally these properties, please see [dangerouslySetAllPagesToNoIndex](#dangerouslySetAllPagesToNoIndex) and [dangerouslySetAllPagesToFollow](#dangerouslySetAllPagesToFollow).
```

Comment:
Small typo: 

`and [dangerouslySetAllPagesToFollow ]` 👉 `and [dangerouslySetAllPagesToNoFollow]`

Reply:
Done 👍